components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/DefaultOIDCSessionStateManager.java

```diff
@@ -94,22 +94,22 @@ public Cookie addOPBrowserStateCookie(HttpServletResponse response) {
      * @param response
      * @param request
      * @param tenantDomain
+     * @param opbsValue
      * @return Cookie
      */
     @Override
     public Cookie addOPBrowserStateCookie(HttpServletResponse response, HttpServletRequest request,
-                                          String tenantDomain) {
+                                          String tenantDomain, String opbsValue) {
```

Comment:
introduce a new method and deprecate this

Reply:
The public method signature changed in these PRs is added with #1622. As we don't have any product release during this time we are changing the method signature without deprecating.